ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController.swift

```diff
@@ -205,6 +205,8 @@ public class BrowserViewController: UIViewController {
   var activeButtonToast: Toast?
   /// An infobar displaying a privacy notice when a search result ad is clicked
   var searchResultAdClickedInfoBar: SearchResultAdClickedInfoBar?
+  /// An infobar displaying a privacy notice when a new tab takeover is viewed
+  var newTabTakeoverInfoBar: NewTabTakeoverInfoBar?
```

Comment:
nit: Infobar

Reply:
Discussed in DM and decided to keep it as it is, because chromium uses `InfoBar`